### ISSUE TITLE
Make README better

### ISSUE THREAD

**Issue by patricia-gallardo** [turtlebrowser/turtlebrowser#39]

**Comment by aminya** [comment]
It would be good to explicitly state what problem Turtule browser is trying to solve.

**Comment by patricia-gallardo** [comment]
Yeah, you're right, I've started with making a separate dev build instructions README for Windows, it is here, but similar instructions need to be made for Linux and Mac and probably the ones for Windows also need to be improved:
https://github.com/turtlebrowser/turtlebrowser/blob/master/BUILD_WINDOWS.md
